src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+
+function Person({
+  person,
+}) {
+  let message = '';
+
+  if (!person.isMarried) {
+    message = 'I am not married';
+  }
+
+  if (person.isMarried && person.sex === 'm') {
+    message = `${person.partnerName} is my wife`;
+  }
+
+  if (person.isMarried && person.sex === 'f') {
+    message = `${person.partnerName} is my husband`;
+  }
```

Comment:
```suggestion
  if (person.sex === 'm') {
    message = `${person.partnerName} is my wife`;
  }

  if (person.sex === 'f') {
    message = `${person.partnerName} is my husband`;
  }
  
  if (!person.isMarried) {
    message = 'I am not married';
  }
```

Reply:
but better to avoid such checks inside component - use helper function to generate massage

Reply:
I created special function for check status of marriage each person